composables/masto/translate.ts

```diff
@@ -103,7 +103,7 @@ export function useTranslation(status: mastodon.v1.Status | mastodon.v1.StatusEd
     && supportedTranslationCodes.includes(to as any)
     && supportedTranslationCodes.includes(status.language as any)
     && !userSettings.value.disabledTranslationLanguages.includes(status.language)
-  const enabled = /*! !useRuntimeConfig().public.translateApi && */ shouldTranslate
```

Comment:
I'm very interested in why this was just a single `!` before lol

Reply:
@jpezninjo oh lol yeah I was wondering that too; but if you look closer the second `!` is there right after the `/*`. How it got there,, i don't know.

Reply:
Ah that makes complete sense. Good eye!